include/spdlog/sinks/qt_sinks.h

```diff
@@ -53,50 +62,184 @@ class qt_sink : public base_sink<Mutex>
     std::string meta_method_;
 };
 
+#if !defined(SPDLOG_USE_STD_FORMAT)
+template <typename Mutex>
+class qt_sink_color : public qt_sink<Mutex> {
+public:
+    qt_sink_color(QObject *qt_object = nullptr, bool color_ = true, const std::string &meta_method = "")
+    {
+        qt_sink<Mutex>::qt_object_ = qt_object;
+        qt_sink<Mutex>::meta_method_ = meta_method;
+        should_do_colors_ = color_;
```

Comment:
Whats the point of shoild_do_colors? If colors not needed this sinks should not be used anyway

Reply:
Ok, I removed this